src/datetime.js

```diff
@@ -555,7 +555,9 @@ export default class DateTime {
    */
   static fromMillis(milliseconds, options = {}) {
     if (!isNumber(milliseconds)) {
-      throw new InvalidArgumentError("fromMillis requires a numerical input");
+      throw new InvalidArgumentError(
+        "fromMillis requires a numerical input, but received " + milliseconds
```

Comment:
If `milliseconds` is a string, like "123", this will be a confusing message. Perhaps more like

```
throw new InvalidArgumentError(
  `fromMillis requires a numerical input, but received a ${typeof milliseconds} with value ${milliseconds}`)
```